app/views/static_pages/landing_page.html.erb

```diff
@@ -27,7 +27,7 @@
 
     <h2 class="govuk-heading-m govuk-!-margin-top-6">How to apply</h2>
 
-    <% if additional_payments_open? %>
+    <% if additional_payments_open?(current_policy) %>
```

Comment:
I think this landing page is specific to the `additional-payments` journey, we may need to consider a different template for TSLR.

Reply:
This was a temporary solution to get these tickets over the line and the plan is to address this properly in the card related to the tslr landing page [CAPT-1353](https://dfedigital.atlassian.net/browse/CAPT-1353) "Dev - TSLR ID - New landing page"